### DISCUSSION TITLE
[Request] "Show 'real' names of staff"

### DISCUSSION THREAD

**Discussion by Krutonium** [OpenRCT2/OpenRCT2#15710] [Feature requests] [answered]
I would love an option to use the same name RNG as the peeps to automatically name my Staff Members.

**Comment by ceeac** [comment]
I'll try working on this either today or tomorrow, if nobody else is working on it already.

**Comment by Liampie** [comment]
Nice idea. What are the possibilities for customising the names available? The possible peep names are all in a database I assume (the combinations of first names and surnames are fixed). I guess the easiest thing would be if the staff would draw the names from this database as well. If it's possible to add entirely new names, that opens up a lot of ideas and possibilities.

In the future it would be great if there was such a thing as 'name packs'. You would select "Germany" as the setting of the park in the scenario settings and all peeps and mechanics would be German. I would love that

**Comment by rwjuk** [comment]
@Liampie Best to submit a separate issue for a separate idea - makes discussion easier to manage.

@ceeac Nobody else is working on this to my knowledge, it's all yours!

**Comment by Gymnasiast** [comment]
You know, it could be nice if the staff were given names never given to guests. How about names like Ted, Duncan, Yaroslav, Michael, Michał, Lewis, Marijn, Richard, and Kenton, Jarno and Rune?
(And perhaps Robert and Hielke.)

**Comment by rwjuk** [comment]
Hehe, nice little easter egg. As long as it wouldn't be a massive pain to implement, and everyone's happy with their name being used, I'd go with it! :)

**Comment by Ryder17z** [comment]
@Gymnasiast I really like this idea

**Comment by Krutonium** [comment]
I'm totally for it :D

And Hypothetically it shouldn't be hard to add - new dictionary of names, and when applying the names, simply rename them as a user would.

**Comment by J-PIE-314** [comment, marked as answer]
This will be implemented in #22732.

**Reply by J-PIE-314** [reply]
The aforementioned pull request has been merged.
